src/components/reusable/modal/modal.stories.js

```diff
@@ -48,6 +50,8 @@ export const Modal = {
         ?destructive=${args.destructive}
         ?okDisabled=${args.okDisabled}
         ?hideFooter=${args.hideFooter}
+        ?hideSecondaryButton=${args.hideSecondaryButton}
+        ?hideCancelButton=${args.hideCancelButton}
```

Comment:
Add `secondaryButtonText=${args.secondaryButtonText}` So that it can be modifiable from storybook controls. 
<img width="1265" alt="Screenshot 2024-04-26 at 4 05 18 PM" src="https://github.com/kyndryl-design-system/shidoka-applications/assets/140634142/ff26c270-d6e0-4735-bf34-6bf821b195ee">